Source/Fuse.Controls.TimePicker/TimePicker.Docs.uno

```diff
@@ -0,0 +1,67 @@
+namespace Fuse.Controls
+{
+	/**
+		Displays a component to select a time.
+
+		Currently, the TimePicker only has native implementations, so it should be contained in a @NativeViewHost.
+
+		A `TimePicker` can be used to select a specific time value. The type of its `Value` property is `Uno.DateTime`,
+		which is marshalled automatically to and from the JavaScript `Date` type. This makes interaction between JavaScript
+		and the `TimePicker` type seamless via databinding. If you plan to wrap a `TimePicker` in a UX component and use a
+		UX property to hook up this value, the `Uno.DateTime` type should be used.
+
+		Both `Uno.DateTime` and JS' `Date` type represent a specific timestamp. These types have both date and time
+		components, and their interpretation depends on a given time zone, which can cause a great deal of confusion. To
+		simplify all of this and ensure consistent behavior accross different time zones and locales, `TimePicker` will assume
+		incoming values are relative to UTC, and truncate the date component to the Unix epoch (1 Jan 1970), effectively
+		ignoring the date component altogether. Similarly, values read from `TimePicker` properties will only consist of a time
+		component at on 1 Jan 1970. This makes values going to/from the `TimePicker` control easy to create and interpret
+		consistently, but also means that if a value with a date component other than the unix epoch is written to TimePicker`'s
+		`Value` property, subsequent values read from the property may not match the written value, as the date component will
+		have been truncated.
+
+		Note that this control should not be used to deal with historic dates, as calendar/date and timestamp translation is
+		inconsistent between different locales due to when and where different calendar systems were adopted. However, the
+		behavior is consistent for all dates since at least 1900 including all representable future dates.
+
+		Certain Android versions (prior to Android 7) which use the clock view by default appear to have some issues with
+		setting the hour/minute of the `TimePicker` programmatically while interacting with the picker, so some care
```

Comment:
I think the wording should be changed to say something like, "You should avoid modifying the TimePicker values programmatically while the control has focus as this is know to have issues on some Android devices."  It just explicitly says what not to do.

Reply:
Yeah, that's certainly more clear. I can change that.